revproxy/views.py

```diff
@@ -38,30 +45,57 @@ def __init__(self, *args, **kwargs):
 
     @property
     def upstream(self):
+        """This method is not implemanted on the base class """
         raise NotImplementedError('Upstream server must be set')
 
     @property
     def diazo_rules(self):
+        """Method to create a path to the file containing the diazo rules
+
+        :returns: Path of diazo rules file
+        """
         child_class_file = sys.modules[self.__module__].__file__
         app_path = os.path.abspath(os.path.dirname(child_class_file))
         return os.path.join(app_path, 'diazo.xml')
 
     @classonlymethod
     def as_view(cls, **initkwargs):
+        """This method return a callable View that take a request and returns a
+        response, but the View will have a Cross-site request forgery verified
+
+        :returns: A callable view
+        """
         view = super(ProxyView, cls).as_view(**initkwargs)
         view.csrf_exempt = True
         return view
 
-    def dispatch(self, request, path):
+    def __format_path_to_redirect(self, request):
+        """This method evaluate if the path request must be redirected and
```

Comment:
by deault python returns None that's already considered False. So, if there is somewhere to redirect we return it, if not we return None. (this is how it is in master, please rebase).